.travis.yml

```diff
@@ -21,23 +21,46 @@ matrix:
       dist: xenial
       sudo: true
       name: "Python 3.7 without Extensions"
+    - env: TOX_ENV=py38
```

Comment:
There has to be a better way of doing this instead of just copy / pase the config block.

Not really a blocker on this PR but just a note for us to look for in the future.